StyleGuide.md

```diff
@@ -1045,12 +1045,16 @@ Sometimes, you need to include files that are maintained in other places than th
 - SQL diagrams, which are maintained in the `cockroachdb/cockroach` repo
 - Code samples, which are maintained in various repos
 
-For SQL diagrams, you remotely include the entire (HTML) file as follows:
+###### SQL diagrams
+  
+For [SQL diagrams](https://cockroachlabs.atlassian.net/wiki/spaces/ED/pages/1134166645/SQL+Grammar+Documentation#Add-a-SQL-diagram-to-a-SQL-reference-page), you remotely include the entire (HTML) file as follows:
```

Comment:
This page requires CRL authentication and is not public. I would consider not hyperlinking to it.

Reply:
I wonder if we should even include this section at all in the public style guide. Do we expect external users to work with SQL diagrams?

Reply:
I feel like we won't rely on the public to craft diagrams. We should probably just keep it in our wiki.

Reply:
my opinion: 99% of the usage of this doc will be by our team and should be optimized for that use case.  We don't get that many external contributions.  Over the years we have probably spent more time talking about things that could hypothetically affect external contributions than actually reviewing said contributions (they are quite rare)